### ISSUE TITLE
Uncaught TypeError: undefined is not a function

### ISSUE THREAD

**Issue by Cartman0** [griffin-stewie/atom-redpen#3]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.2
**System**: Unknown Windows Version
**Thrown From**: [redpen](https://github.com/griffin-stewie/atom-redpen) package, v0.3.3

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At C:\Users\nabana\.atom\packages\redpen\lib\redpenValidator.coffee:26

TypeError: undefined is not a function
  at Validator.module.exports.Validator.needsValidate (C:\Users\nabana\.atom\packages\redpen\lib\redpenValidator.coffee:26:31)
  at Validator.module.exports.Validator.versionCheck (C:\Users\nabana\.atom\packages\redpen\lib\redpenValidator.coffee:40:15)
  at Object.module.exports.Redpen.validate (C:\Users\nabana\.atom\packages\redpen\lib\redpen.coffee:70:16)
  at atom-workspace.subscriptions.add.atom.commands.add.redpen:validate (C:\Users\nabana\.atom\packages\redpen\lib\redpen.coffee:40:83)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\nabana\AppData\Local\atom\app-1.0.2\resources\app.asar\src\command-registry.js:241:29)
  at CommandRegistry.handleCommandEvent (C:\Users\nabana\AppData\Local\atom\app-1.0.2\resources\app.asar\src\command-registry.js:3:61)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (C:\Users\nabana\AppData\Local\atom\app-1.0.2\resources\app.asar\src\command-registry.js:156:19)
  at EventEmitter.<anonymous> (C:\Users\nabana\AppData\Local\atom\app-1.0.2\resources\app.asar\src\window-event-handler.js:70:30)
  at emitOne (events.js:77:13)
  at EventEmitter.emit (events.js:166:7)

```

### Commands

```
     -4:04.1.0 application:add-project-folder (ul.list-inline.tab-bar.inset-panel)
     -3:58.6.0 linter:set-bubble-transparent (atom-text-editor.editor.is-focused)
     -3:56.5.0 key-binding-resolver:toggle (atom-text-editor.editor.is-focused)
     -3:50.8.0 application:open-file (atom-text-editor.editor.is-focused)
  3x -3:46.8.0 core:backspace (atom-text-editor.editor.is-focused)
  2x -3:45.4.0 linter:set-bubble-transparent (atom-text-editor.editor.is-focused)
     -3:43 window:run-package-specs (atom-text-editor.editor.is-focused)
     -0:10.5.0 redpen:validate (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-light-ui",
      "atom-light-syntax"
    ],
    "disabledPackages": [
      "autocomplete-paths",
      "turbo-javascript",
      "linter-cpplint",
      "linter-tidy"
    ]
  },
  "redpen": {
    "pathForRedPen": "C:\\Users\\nabana\\redpen-cli-1.2\\bin\\redpen",
    "validateOnSave": true,
    "JAVA_HOME": "C:\\Program Files (x86)\\Java\\jre1.8.0_45\\bin",
    "pathForConfigurationXMLFile": "C:\\Users\\nabana\\redpen-cli-1.2\\conf\\redpen-conf-ja.xml"
  }
}
```

### Installed Packages

```coffee
# User
color-picker, v2.0.9
emmet, v2.3.12
file-icons, v1.5.8
javascript-snippets, v1.0.0
linter, v1.2.4
linter-csslint, v0.0.14
linter-htmlhint, v0.0.17
linter-js-standard, v1.2.0
minimap-color-highlight, v4.1.4
project-manager, v1.15.11
redpen, v0.3.3
Sublime-Style-Column-Selection, v1.3.0
web-browser, v1.5.0

# Dev
No dev packages
```

**Comment by tos-kamiya** [comment]
I've got a similar error.

[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.0
**System**: Ubuntu 15.04
**Thrown From**: [redpen](https://github.com/griffin-stewie/atom-redpen) package, v0.3.3


### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /home/toshihiro/.atom/packages/redpen/lib/redpenValidator.coffee:26

TypeError: undefined is not a function
    at Validator.module.exports.Validator.needsValidate (/home/toshihiro/.atom/packages/redpen/lib/redpenValidator.coffee:49:31)
    at Validator.module.exports.Validator.versionCheck (/home/toshihiro/.atom/packages/redpen/lib/redpenValidator.coffee:60:17)
    at Object.module.exports.Redpen.validate (/home/toshihiro/.atom/packages/redpen/lib/redpen.coffee:75:29)
    at atom-workspace.subscriptions.add.atom.commands.add.redpen:validate (/home/toshihiro/.atom/packages/redpen/lib/redpen.coffee:47:26)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/opt/atom/resources/app.asar/src/command-registry.js:241:29)
    at /opt/atom/resources/app.asar/src/command-registry.js:3:61
    at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/opt/atom/resources/app.asar/node_modules/command-palette/lib/command-palette-view.js:159:32)
    at CommandPaletteView.module.exports.SelectListView.confirmSelection (/opt/atom/resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
    at space-pen-li.<anonymous> (/opt/atom/resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:139:19)
    at HTMLOListElement.jQuery.event.dispatch (/opt/atom/resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
```

### Commands

```
     -0:04.4.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:00.8.0 redpen:validate (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "themes": [
      "one-light-ui",
      "one-light-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
file-icons, v1.5.8
japanese-wrap, v0.2.10
language-latex, v0.6.1
language-rust, v0.4.3
redpen, v0.3.3
tool-bar, v0.1.8
tool-bar-main, v0.0.8

# Dev
No dev packages
```

**Comment by griffin-stewie** [comment]
@Cartman0 @tos-kamiya 
Thank you guys for reporting. I could reproduce the issue. I am going to try to fix it.

**Comment by griffin-stewie** [comment]
@Cartman0 @tos-kamiya 
I released v0.3.4 that I fixed this issue.
